en/docs/develop/tools/migration-tools/migrate-from-mulesoft.md

```
@@ -109,25 +114,62 @@ While the agent is running:
-
 </TabItem>
-<TabItem value="code" label="Ballerina Code">
-
-```bash
-# Migrate a MuleSoft project
-bal migrate mule -i /path/to/mule-project/ -o migrated/
+<TabItem value="code" label="CLI">
+
+You can migrate MuleSoft projects using the Ballerina CLI tool. Follow these steps:
+
+### Prerequisite
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)